infrastructure/variables.tf

```diff
@@ -5,7 +5,7 @@ variable "product" {
 
 variable "microservice" {
   type    = "string"
-  default = "job-scheduler"
+  default = "sscs-job-scheduler"
```

Comment:
the microservice name hasn't changed just the product name

Reply:
It's going to be a new micro service though, Job-Scheduler will be a different app

Reply:
Updating now to match our sscs product names